telemetry/telemetry_record.py

```diff
@@ -1,23 +1,30 @@
-"""Record WebPageReplay
+"""Record WebPageReplay, modify, and record page load times
 
 Records running a set of target urls on the remote tablet, modifies cacheable
 objects, then re-runs to see page load time differences.
 
-Usage: sudo python record_wpr.py
+Usage: sudo python telemetry_record.py
 
 Notes:
     Assumes - telemetry is in CHROMIUM_SRC/tools/telemetry/telemetry
             - page_sets is in CHROMIUM_SRC/tools/perf/page_sets
             - modify_wpr_delays.py is in CHROMIUM_SRC/third_party/webpagereplay
 
-TODO: Measure page load time before and after modifying
 TODO: Make sure to only get the most recent wpr file when moving / modifying
 TODO: Clean up path dependencies, move them to the top
 """
+from base64 import b64encode
+from os import listdir, path
+from re import findall
 from subprocess import Popen, PIPE, STDOUT
 from sys import path
+import cPickle
+import json
 import os
 import pickle
```

Comment:
Yeah, right now I use pickle to store temporary data for each trial and the aggregate data for everything. The reasoning for this is that if this program crashes, there will still be evidence of the urls that ran. I also use pickle files to let this program and benchmarks communicate data.
I use cPickle to read the wpr files.